lib/MusicBrainz/Server/Controller/ReleaseEditor.pm

```diff
@@ -43,6 +43,7 @@ sub _init_release_editor
 
     my @medium_formats = $c->model('MediumFormat')->get_all;
     my $discid_formats = [ grep { $_ } map { $_->has_discids ? ($_->id) : () } @medium_formats ];
+    my %format_dates = map { $_->id => $_->year } @medium_formats;
```

Comment:
Would it make sense to fall back to the parent/root format's year if a specific one isn't set?

Reply:
I'm not sure I'd dare to block by inheritance - I'm sure there'll be some obscure case where that doesn't work. Honestly, the main use case for this is just blocking CDs before the 1980s, so either way would work.

Reply:
It seems weird that the error basically never triggers for any of the child formats even when it obviously should, but they're probably rare enough that it doesn't matter.